core/components/molecules/_action-input/_action-input.js

```diff
@@ -12,6 +12,13 @@ import { actionShape } from '@auth0/cosmos/_helpers/action-shape'
 /* TODO: width of button should be exported by button component */
 const widthOfButton = '36px'
 
+export const topMarginForActions = {
```

Comment:
last one, I promise 😂 Should this be imported from `components/_helpers/action-spacing.js` instead of replicated?

Reply:
Actually, I removed that file because we were only using it here 😛